tests/Client.UiTests/Tests.fs

```diff
@@ -0,0 +1,42 @@
+module Client.UiTests.Tests
+
+open canopy.classic
+open Expecto
+open System.IO
+
+let serverUrl = "http://localhost:8080/"
+
+type TypeInThisAssembly() =
+    class end
+
+let screenshotFolder = FileInfo(System.Reflection.Assembly.GetAssembly(typeof<TypeInThisAssembly>).Location).Directory.FullName
+
+let testCase name f =
+    testCase name (fun x ->
+        try
+            f x
+        with
+        | exn ->
+            screenshot screenshotFolder (name  + "-" + System.DateTime.Now.ToString("MMM-d_HH-mm-ss")) |> ignore
+            raise exn
+    )
+
+let startApp () =
+    url serverUrl
+    waitForElement("#elmish-app")
+
+let tests =
+    testList "client tests" [
+        testCase "sound check - server is online" (fun () ->
+            startApp ()
```

Comment:
So that's like starting app for every test?

Reply:
that's more like opening the app in browser. server/client are not stopped between tests

Reply:
I see, how about renaming to `openApp`?